supervisor/supervisor.go

```diff
@@ -275,6 +272,9 @@ func (s *Supervisor) Start() error {
 		"memory":      s.machine.Memory,
 		"cpus":        s.machine.Cpus,
 	}).Debug("containerd: supervisor running")
+	if err := s.restore(); err != nil {
```

Comment:
small nit, I'd rather this be done before we print that the supervisor is running.